include/elliptics/packet.h

```diff
@@ -340,30 +359,67 @@ struct dnet_backend_control
 struct dnet_id_container
 {
 	int backends_count;
-	struct dnet_backend_ids backends[0];
 } __attribute__ ((packed));
 
-static inline int dnet_validate_id_container(struct dnet_id_container *ids, size_t size, struct dnet_backend_ids **backends)
+static inline void dnet_convert_id_container(struct dnet_id_container *cnt)
```

Comment:
I put every command and data structure which is part of the protocol into public headers because one might want to implement Elliptics client in a different language. No one tried so far, but it does not matter.

This pull request does not and will not change how our commands and structures are stored in the source code.
If you think that half of them has to go into private headers, we can discuss it, most likely I will not accept it because of the reason above. And this whole pull request is not about it. Even this chunk is not about it, it fixes 2 bugs - le/be conversion and possible memory corruption. This could not be done without changing container parser function, and this bug was only visible in server-send test, since it starts servers in parallel and immediately starts working with several groups.